spec/spec_helper.rb

```diff
@@ -5,8 +5,14 @@
 require 'fakeweb'
 require 'pry'
 require 'rspec'
+
 require 'simplecov'
-SimpleCov.start
+require 'coveralls' if ENV['CI']
+
+SimpleCov.formatter = Coveralls::SimpleCov::Formatter if ENV['CI']
+SimpleCov.start do
+  add_filter '/spec/'
+end
```

Comment:
Not sure about the profiles benefits.  Likely none unless the specs need to restart simplecov at some point with a different profile.  I tend to be generous with filters that exclude anything outside the project (like included gems in vendor or whatever), so that can reduce the coveralls %coverage the first time it's run with new filters.